packages/backend-common/src/reading/FetchUrlReader.ts

```diff
@@ -60,11 +69,29 @@ export class FetchUrlReader implements UrlReader {
               }
             : (_url: URL) => true;
           const host = allowConfig.getString('host');
-          if (host.startsWith('*.')) {
-            const suffix = host.slice(1);
-            return (url: URL) => url.host.endsWith(suffix) && checkPath(url);
+          const [hostname, port] = host.split(':');
+
+          let checkPort = (url: URL) => !url.port;
+          if (port) {
+            const isRange = port.includes('-');
+            const range = port.split('-').map(v => parseInt(v, 10)) as [
+              number,
+              number | undefined,
+            ];
+
+            // it allows port ranges ex: :1-200 or :30-
```

Comment:
I'm feeling it might be safest to skip the `30-` format, or what do you think? I'm worried it'll allow reading from upper ports unintentionally